src/pages/fr/guides/environment-variables.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Utiliser des variables d'environnement
+description: Apprenez comment utiliser les variables d'environnement dans un projet Astro.
+setup: |
+  import ImportMetaEnv from '~/components/ImportMetaEnv.astro';
+---
+
+Astro utilise Vite pour les variables d'environnement, et permet d'utiliser [n'importe quelle de ses méthodes](https://vitejs.dev/guide/env-and-mode.html) pour obtenir et définir des variables d'environnement.
```

Comment:
Astro utilise Vite pour les variables d'environnement, et permet d'utiliser [n'importe quelle méthode de Vite](https://vitejs.dev/guide/env-and-mode.html) pour obtenir et définir des variables d'environnement.